backend/src/test/app.test.ts

```diff
@@ -16,201 +16,308 @@ afterEach(() => {
     jest.clearAllMocks();
 });
 
-describe("Post Election", () => {
+// describe("Post Election", () => {
```

Comment:
Why are all the other test cases commented out?

Reply:
ah mb, I was trying to test them individually forgot to uncomment. I've since learned better ways to run specific tests

When I uncommented the tests, some of them were failing, but I don't think they're related to changes in this PR. More details here https://github.com/Equal-Vote/star-server/issues/74